### ISSUE TITLE
Topic 2 graphics

### ISSUE THREAD

**Issue by hrlee1202** [RCinbo/IPBESValueAssessment#20]

**Comment by hrlee1202** [comment]
Checking 'other' answers in each question - whether we need to check in detail. 
- Scale
- Habitat
- ES list

**Comment by hrlee1202** [comment]
Need checking tables

2.2 The application assesses the following habitats (multiple possible):
[s3_single_with_other_habitat_n127.xlsx](https://github.com/RCinbo/IPBESValueAssessment/files/5405573/s3_single_with_other_habitat_n127.xlsx)

2.10 Other answers on 'regarding Nature itself' 
[s3_single_with_other_nature_n20.xlsx](https://github.com/RCinbo/IPBESValueAssessment/files/5405576/s3_single_with_other_nature_n20.xlsx)

2.11 Other answers on Regulating Nature Contribution to People
[s3_single_with_other_regul_n46.xlsx](https://github.com/RCinbo/IPBESValueAssessment/files/5405578/s3_single_with_other_regul_n46.xlsx)

2.12 Other answers to Material Nature Contribution to People
[s3_single_with_other_material_n25.xlsx](https://github.com/RCinbo/IPBESValueAssessment/files/5405574/s3_single_with_other_material_n25.xlsx)

2.13 Other answers to Non-Material Nature Contribution to People
[s3_single_with_other_non_material_n22.xlsx](https://github.com/RCinbo/IPBESValueAssessment/files/5405577/s3_single_with_other_non_material_n22.xlsx)

2.14 Other answers to Quality of Life
[s3_single_with_other_QoL_n29.xlsx](https://github.com/RCinbo/IPBESValueAssessment/files/5406781/s3_single_with_other_QoL_n29.xlsx)

**Comment by hrlee1202** [comment]
How many NCP were studied per application? (test plot)
![Rplot04](https://user-images.githubusercontent.com/47192753/96551086-16260080-12b2-11eb-843f-1b39d9d27cd1.png)

How many NCP categories were studied per application? (test plot)
![Rplot02](https://user-images.githubusercontent.com/47192753/96553708-d6611800-12b5-11eb-9819-61a10cbecb34.png)

**Comment by Sander-Jacobs** [comment]
habitat table corrected !!

**Comment by Sander-Jacobs** [comment]
2.10 done

**Comment by Sander-Jacobs** [comment]
2.11 done

**Comment by Sander-Jacobs** [comment]
2.12 done

**Comment by Sander-Jacobs** [comment]
2.13 done

**Comment by Sander-Jacobs** [comment]
and 2.14!! all in the folder: https://drive.google.com/drive/folders/12zJIp9cXec2V0UpDxYo8aoIvpmG0AeqG

**Comment by hrlee1202** [comment]
Habitat corrected result

[Fig_habitat_corrected_25Oct.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5442140/Fig_habitat_corrected_25Oct.pdf)

![Fig_habitat_corrected_25Oct](https://user-images.githubusercontent.com/47192753/97241599-e1ee9a80-17f1-11eb-884b-02a5c94d3316.png)

**Comment by Sander-Jacobs** [comment]
the scales table is also corrected by raphael!!

**Comment by hrlee1202** [comment]
"Maintenance of options" belongs to both ''targets of valuation' regarding Nature Itself' (question 2.10) and ''targets of valuation' regarding Regulating Nature Contributions To People' (question 2.11).. Any idea?

**Comment by Sander-Jacobs** [comment]
yes i put this in both categories, but they are the same (in between
categories)
so, the scores can be merged. for representation, i would put them in NCP
class.

**Comment by hrlee1202** [comment]
In the given answers (2.13: non-material NCP), there is no "Living well in harmony with nature". I am not sure why you included in the corrected form?

**Comment by hrlee1202** [comment]
(I accidentally closed the issue and reopen it, Sorry!)

**Comment by hrlee1202** [comment]
[Fig_FullNCP_30Oct.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5462330/Fig_FullNCP_30Oct.pdf)
![Fig_FullNCP_30Oct](https://user-images.githubusercontent.com/47192753/97648011-f7fb9580-1a53-11eb-9ff1-b906e0b34ef9.png)

**Comment by hrlee1202** [comment]
Habitat figure revised

![Fig_habitat_corrected_30Oct](https://user-images.githubusercontent.com/47192753/97648595-79075c80-1a55-11eb-8c19-7129444fdba1.png)
[Fig_habitat_corrected_30Oct.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5462397/Fig_habitat_corrected_30Oct.pdf)

**Comment by Sander-Jacobs** [comment]
It is one of the choice options. wasn't it ticked at all by anyone but me??
that would be curious...

**Comment by hrlee1202** [comment]
The figure I showed last time (Corrected Habitat) was wrong. I additionally put the 'other' again..  I want to check the code before sharing the figures.

**Comment by hrlee1202** [comment]
Hi @Sander-Jacobs

I deleted this comment but it seems sent right away to you. Sorry!  Yes, I realised that It is under 'Quality of Life', you probably accidentally colour-coded it to 'non-material' in the excel file (column colour). that was why I was confused at once, but realised it later - All fine! :-)

**Comment by hrlee1202** [comment]
Number of scales (whether the application considers multiple scales):  Mostly a single scale
[Fig_NumberofScalesPerApp.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5484754/Fig_NumberofScalesPerApp.pdf)

**Comment by hrlee1202** [comment]
2.1 Corrected Scale 
[Fig_2.1_scale_corrected.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5491217/Fig_2.1_scale_corrected.pdf)
Scale with the MF information
[Fig_2.1_scale_corrected_MF.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5491231/Fig_2.1_scale_corrected_MF.pdf)

2.2 Correcred Habitat
[Fig_habitat_corrected_4Nov.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5491257/Fig_habitat_corrected_4Nov.pdf)

2.7 The application collected data with the following temporal frequency:
[Fig_2.7_temporal frequency.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5484771/Fig_2.7_temporal.frequency.pdf)

2.8. The application assesses values change over time?
[Fig_2.8_overTime.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5484783/Fig_2.8_overTime.pdf)

2.9 The application assesses following temporal changes:
[Fig_2.9_overTime.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5484812/Fig_2.9_overTime.pdf)

2.10-2.15 NCP list
[Fig_FullNCP_4Nov.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5491260/Fig_FullNCP_4Nov.pdf)

2.10 NCP Nature with MF
[Fig_nature_MF.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5491261/Fig_nature_MF.pdf)

2.11 Regulating NCP with MF
[Fig_regult_MF.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5491264/Fig_regult_MF.pdf)

2.12. Material NCP with MF
[Fig_material_MF.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5491267/Fig_material_MF.pdf)

2.13 Non-Material NCP with MF
[Fig_non_material_MF.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5491271/Fig_non_material_MF.pdf)

2.14 Quality of Life with MF
[Fig_QoL_MF.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5491272/Fig_QoL_MF.pdf)

How many different NCP were studied per application
[howmanyNCPperApp.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5491281/howmanyNCPperApp.pdf)

How many different NCP categories were studied per application
[howmanyNCPCategoriesperApp.pdf](https://github.com/RCinbo/IPBESValueAssessment/files/5491282/howmanyNCPCategoriesperApp.pdf)
